bot/.env.example

```diff
@@ -0,0 +1,20 @@
+DATABASE_URL="postgresql://airhorn:mysecurepassword@127.0.0.1:5432/airhorn?schema=public"
+SHADOW_DATABASE_URL="postgresql://airhorn:mysecurepassword@127.0.0.1:5432/airhorn_shadow?schema=public"
+
+MIN_LOG_LEVEL="2"
+
+DISCORD_TOKEN=""
+DISCORD_STATUS="airhorn.solutions"
+
+AIRHORN_MAX_QUEUE_ITEMS="3"
+
+DISCORD_SHARD_AUTO="true"
+DISCORD_SHARD_IDS="0,1,2,3"
+DISCORD_SHARD_TOTAL="4"
+
+REGISTER_DISCORD_COMMANDS="false"
+
+WEB_PORT="3000"
+WEB_HOST_STATIC="true"
+WEB_HOST_STATIC_DIRECTORY="../website/build"
+WEB_API_KEY="mysecureapikey"
```

Comment:
Let's add a bonus new line to make the angry red circle line thing go away

Reply:
No more angry red circle line thing. ⛔